plugin.audio.rne/addon.xml

```diff
@@ -20,6 +20,6 @@
         <license>GNU GENERAL PUBLIC LICENSE. Version 3, 29 June 2007</license>
```

Comment:
GPL-3.0-only or GPL-3.0-or-later depending on your license (SPDX identifier please). See https://spdx.org/licenses/

Reply:
OK, I'll fix it. Thanks for the advice.